charts/prometheus-kafka-exporter/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "v1.3.1"
+appVersion: "v1.3.2"
```

Comment:
appVersion should be 1.4.1 as set in values.yaml

Reply:
of course, will do.